lexer/lexer.go

```diff
@@ -21,10 +21,15 @@ func New(input string) *Lexer {
 	l.readChar()
 	l.FSM = fsm.NewFSM(
 		"initial",
+		/*
+			Initial state is default state
+			Nosymbol state support we identify tok ':' is symbol or json
```

Comment:
I think this sentence could be more expressive if we change it to:
`"Nosymbol state helps us identify tok ':' is for symbol or hash value"`
(`{ key:value}` this is not a json, it's a hash)